e2e/cypress/integration/ui-smoke/ui-create-delete-metadata.spec.js

```diff
@@ -10,21 +10,14 @@ describe('Adds metadata to a sample in a created project', () => {
   //   2. Log in into biomage
   //   3. Visit data-management
   beforeEach(() => {
-    // Intercept PUT/DELETE calls to */project/* endpoint
+    // Intercept PUT calls to */project/* endpoint
     cy.intercept(
       {
         method: 'PUT',
         url: '*/projects/*',
       },
     ).as('putProject');
 
-    cy.intercept(
-      {
-        method: 'DELETE',
-        url: '*/projects/*',
-      },
-    ).as('deleteProject');
-
```

Comment:
Why is this deleted?

Reply:
Because we aren't using the delete method at any point in this test, we only care about the update calls